posts/2024-07-09-pathogfair-preprint.md

```diff
@@ -0,0 +1,58 @@
+---
+media:
+  - mastodon-eu-freiburg
+  - matrix-eu-announce
+
+mentions:
+  mastodon-eu-freiburg:
+    - bebatut@piaille.fr 
+
+hashtags:
+  mastodon-eu-freiburg:
+    - metagenomics
+    - pathogen
+    - usegalaxy
+
+---
+
+**1/6 🚀 Exciting news in pathogen detection!**  
```

Comment:
```suggestion
**🚀 Exciting news in pathogen detection!**  
```

As you see in the preview we add the 1/6 automatically in medias where is makes sense. Maston we add 1/6 but on Matrix we don't.
Does that make sense?